test/runnable/objc_instance_variable.d

```diff
@@ -0,0 +1,45 @@
+// EXTRA_OBJC_SOURCES: objc_instance_variable.m
+// REQUIRED_ARGS: -L-framework -LFoundation
+
+extern (Objective-C) class NSObject {}
+
+// Defined in `runnable/extra-files/objc_instance_variable.m`
+extern (Objective-C)
+class Foo : NSObject
+{
+    // int a;
+    // int b;
+    // int c;
+
+    // Intentionally not declared the above instance variables here to simulate
+    // that the base class has changed.
+}
+
+extern (Objective-C)
+class Bar : Foo
+{
+    int d;
+
+    static Bar alloc() @selector("alloc");
+    Bar init() @selector("init");
+    void release() @selector("release");
+
+    void bar() @selector("bar") {}
+}
+
+// This is implemented in `runnable/extra-files/objc_instance_variable.m` and
+// returns the value of instance variable `c`.
+extern (C) int getInstanceVariable(Foo);
+
+import std.stdio;
+
+void main()
+{
+    auto bar = Bar.alloc.init;
+    scope (exit) bar.release();
+    bar.d = 3;
```

Comment:
I would set all the values at a non-zero value, not only `c`.

Reply:
1. You mean not only **d**?
2. Do you mean to set all instance variables on the Objective-C side?

Reply:
Erg, sorry. Yes.

Basically the idea is that you could be getting `a`, `b` or `c` with your test, because all 3 of them are set to 0.
I trust it works right now, but if a later change breaks it (e.g. ob1 error) it might not show up.

Reply:
Fixed.